doc/cephadm/services/monitoring.rst

```diff
@@ -83,6 +83,29 @@ steps below:
 
      ceph orch apply grafana
 
+.. _cephadm-monitoring-centralized-logs:
+
+Centralized Logging in Ceph
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ceph now provides centralized logging solution using Loki & Promtail. Centralized Log Management (CLM) is a type
+of logging solution system that consolidates all of your log data and pushes it to one central, accessible,
+and easy-to-use interface. Centralized logging is designed to make your life easier.
+Some of the advantages of using centralized logging are:
```

Comment:
Ceph now provides centralized logging with Loki & Promtail. Centralized Log Management (CLM) consolidates all  log data and pushes it to a central repository, with an accessible and easy-to-use interface. Centralized logging is designed to make your life easier. Some of the advantages are:

Reply:
Thanks @anthonyeleven , Updated the PR